src/components/Form/Form.styles.js

```diff
@@ -1,6 +1,10 @@
 import styled from '@emotion/styled';
 import { Link } from 'react-router-dom';
 
+export const FormContent = styled.form`
+  ${(props) => props.className};
```

Comment:
should it be called `className` if it's used this way? maybe `formStyle` instead? @mollerjorge thoughts?

Reply:
it's consistent with the conventional naming -- it will be easy to understand what it's for. Although I don't have strong opinions on this.

Reply:
I would go with something like `formStyles` or just `styles`

Reply:
I don't have a strong opinion about this either, but I agree it might bit more clear to read props.formStyles than props.className tho.